pyproject.toml

```diff
@@ -40,6 +42,10 @@ implicit_reexport = true
 allow_untyped_decorators = true
 warn_return_any = false
 
+[[tool.mypy.overrides]]
```

Comment:
![image](https://user-images.githubusercontent.com/18153319/182481845-0cfba2db-dcb2-4040-9bfb-69d3a85da268.png)

Reply:
Deleted it 1 minute ago))